### PR TITLE
Refactored MD030 issues

### PR DESCRIPTION
**Pull request by drpayyne**
## Purpose of this pull request

This pull request (PR) refactors Markdown linting: spaces after list markers (MD030) rule in the folders:

- `guides/v2.2/release-notes`

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md [hunk 1]
```diff
@@ -14,49 +14,49 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
+*  Improvements to our core bundled extensions.
 
 Looking for more information on these new features as well as many others? Check out [Magento 2.2.x Developer Documentation]({{ site.baseurl }}/guides/v2.2/) and the [Magento Open Source User Guide](http://docs.magento.com/m2/ce/user_guide/getting-started.html).
 
 ### Core code highlights
 
 This release includes significant performance improvements to the core Magento code:
 
-* <!--- MAGETWO-80789  MAGETWO-88808  MAGETWO-89545 -->  Merchants can now  run the catalog search full text indexer and category product indexer in parallel mode by store view, which can significantly decrease  `indexer:reindex` execution time when running Magento with multiple store views and shared catalogs.
+*  <!--- MAGETWO-80789  MAGETWO-88808  MAGETWO-89545 -->  Merchants can now  run the catalog search full text indexer and category product indexer in parallel mode by store view, which can significantly decrease  `indexer:reindex` execution time when running Magento with multiple store views and shared catalogs.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md]:
```suggestion
*  <!--- MAGETWO-80789  MAGETWO-88808  MAGETWO-89545 -->Merchants can now  run the catalog search full text indexer and category product indexer in parallel mode by store view, which can significantly decrease  `indexer:reindex` execution time when running Magento with multiple store views and shared catalogs.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md [hunk 2]
```diff
@@ -14,49 +14,49 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!--- MAGETWO-74154 --> Refactoring of the catalog full text indexer has improved indexing performance up to 15% for very large profiles (600,000 products) and product catalogs with many configurable options (5,000 configurable products and 500 options).
+*  <!--- MAGETWO-74154 --> Refactoring of the catalog full text indexer has improved indexing performance up to 15% for very large profiles (600,000 products) and product catalogs with many configurable options (5,000 configurable products and 500 options).
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md]:
```suggestion
*  <!--- MAGETWO-74154 -->Refactoring of the catalog full text indexer has improved indexing performance up to 15% for very large profiles (600,000 products) and product catalogs with many configurable options (5,000 configurable products and 500 options).
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md [hunk 3]
```diff
@@ -14,49 +14,49 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!--- MAGETWO-88775 -->  Improving the behavior of swatch product attributes has improved search result page performance up to 31% for catalogs with many configurable product options (for example, 5,000 configurable products and 500 options).
+*  <!--- MAGETWO-88775 -->  Improving the behavior of swatch product attributes has improved search result page performance up to 31% for catalogs with many configurable product options (for example, 5,000 configurable products and 500 options).
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md]:
```suggestion
*  <!--- MAGETWO-88775 -->Improving the behavior of swatch product attributes has improved search result page performance up to 31% for catalogs with many configurable product options (for example, 5,000 configurable products and 500 options).
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md [hunk 4]
```diff
@@ -14,49 +14,49 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
 ### Community contribution highlights
 
 Highlights of community contributions include  fixes that improve checkout flow and the sorting of simple products:
 
-* <!--- MAGETWO-87115 --> Customers can now create an account from the Order Confirmation page. Previously, a customer could not populate the required fields to create an account from this page, and Magento displayed an error.
+*  <!--- MAGETWO-87115 --> Customers can now create an account from the Order Confirmation page. Previously, a customer could not populate the required fields to create an account from this page, and Magento displayed an error.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md]:
```suggestion
*  <!--- MAGETWO-87115 -->Customers can now create an account from the Order Confirmation page. Previously, a customer could not populate the required fields to create an account from this page, and Magento displayed an error.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md [hunk 5]
```diff
@@ -14,49 +14,49 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!--- MAGETWO-73479 --> Magento now correctly applies coupon codes that exclude bundle products. Previously, Magento applied these coupons but did not exclude bundle products as expected.
+*  <!--- MAGETWO-73479 --> Magento now correctly applies coupon codes that exclude bundle products. Previously, Magento applied these coupons but did not exclude bundle products as expected.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md]:
```suggestion
*  <!--- MAGETWO-73479 -->Magento now correctly applies coupon codes that exclude bundle products. Previously, Magento applied these coupons but did not exclude bundle products as expected.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md [hunk 6]
```diff
@@ -14,49 +14,49 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!-- MAGETWO-86227 --> When sorting simple products, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
+*  <!-- MAGETWO-86227 --> When sorting simple products, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md]:
```suggestion
*  <!-- MAGETWO-86227 -->When sorting simple products, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md [hunk 7]
```diff
@@ -14,49 +14,49 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!-- MAGETWO-73419 --> When sorting simple products with a required custom option, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
+*  <!-- MAGETWO-73419 --> When sorting simple products with a required custom option, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5CE.md]:
```suggestion
*  <!-- MAGETWO-73419 -->When sorting simple products with a required custom option, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md [hunk 8]
```diff
@@ -14,39 +14,39 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
+*  Improvements to our core bundled extensions.
 
 Looking for more information on these new features as well as many others? Check out [Magento 2.2.x Developer Documentation]({{ site.baseurl }}/guides/v2.2/) and the [Magento Commerce User Guide](http://docs.magento.com/m2/ee/user_guide/getting-started.html).
 
 ### Core code highlights
 
 This release includes significant performance improvements to the core Magento code:
 
-* <!--- MAGETWO-80789  MAGETWO-88808  MAGETWO-89545 -->  Merchants can now  run the catalog search full text indexer and category product indexer in parallel mode by store view, which can significantly decrease  `indexer:reindex` execution time when running Magento with multiple store views and shared catalogs.
+*  <!--- MAGETWO-80789  MAGETWO-88808  MAGETWO-89545 -->  Merchants can now  run the catalog search full text indexer and category product indexer in parallel mode by store view, which can significantly decrease  `indexer:reindex` execution time when running Magento with multiple store views and shared catalogs.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md]:
```suggestion
*  <!--- MAGETWO-80789  MAGETWO-88808  MAGETWO-89545 -->Merchants can now  run the catalog search full text indexer and category product indexer in parallel mode by store view, which can significantly decrease  `indexer:reindex` execution time when running Magento with multiple store views and shared catalogs.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md [hunk 9]
```diff
@@ -14,39 +14,39 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!--- MAGETWO-74154 --> Refactoring of the catalog full text indexer has improved indexing performance up to 15% for very large profiles (600,000 products) and product catalogs with many configurable options (5,000 configurable products and 500 options).
+*  <!--- MAGETWO-74154 --> Refactoring of the catalog full text indexer has improved indexing performance up to 15% for very large profiles (600,000 products) and product catalogs with many configurable options (5,000 configurable products and 500 options).
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md]:
```suggestion
*  <!--- MAGETWO-74154 -->Refactoring of the catalog full text indexer has improved indexing performance up to 15% for very large profiles (600,000 products) and product catalogs with many configurable options (5,000 configurable products and 500 options).
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md [hunk 10]
```diff
@@ -14,39 +14,39 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!--- MAGETWO-88775 -->  Improving the behavior of swatch product attributes has improved search result page performance up to 31% for catalogs with many configurable product options (for example, 5,000 configurable products and 500 options).
+*  <!--- MAGETWO-88775 -->  Improving the behavior of swatch product attributes has improved search result page performance up to 31% for catalogs with many configurable product options (for example, 5,000 configurable products and 500 options).
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md]:
```suggestion
*  <!--- MAGETWO-88775 -->Improving the behavior of swatch product attributes has improved search result page performance up to 31% for catalogs with many configurable product options (for example, 5,000 configurable products and 500 options).
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md [hunk 11]
```diff
@@ -14,39 +14,39 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
 ### Community contribution highlights
 
 Highlights of community contributions include  fixes that improve checkout flow and the sorting of simple products:
 
-* <!--- MAGETWO-87115 --> Customers can now create an account from the Order Confirmation page. Previously, a customer could not populate the required fields to create an account from this page, and Magento displayed an error.
+*  <!--- MAGETWO-87115 --> Customers can now create an account from the Order Confirmation page. Previously, a customer could not populate the required fields to create an account from this page, and Magento displayed an error.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md]:
```suggestion
*  <!--- MAGETWO-87115 -->Customers can now create an account from the Order Confirmation page. Previously, a customer could not populate the required fields to create an account from this page, and Magento displayed an error.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md [hunk 12]
```diff
@@ -14,39 +14,39 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!--- MAGETWO-73479 --> Magento now correctly applies coupon codes that exclude bundle products. Previously, Magento applied these coupons but did not exclude bundle products as expected.
+*  <!--- MAGETWO-73479 --> Magento now correctly applies coupon codes that exclude bundle products. Previously, Magento applied these coupons but did not exclude bundle products as expected.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md]:
```suggestion
*  <!--- MAGETWO-73479 -->Magento now correctly applies coupon codes that exclude bundle products. Previously, Magento applied these coupons but did not exclude bundle products as expected.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md [hunk 13]
```diff
@@ -14,39 +14,39 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!-- MAGETWO-86227 --> When sorting simple products, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
+*  <!-- MAGETWO-86227 --> When sorting simple products, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md]:
```suggestion
*  <!-- MAGETWO-86227 -->When sorting simple products, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md [hunk 14]
```diff
@@ -14,39 +14,39 @@ See [Magento Security Center](https://magento.com/security/patches/magento-2.2.5
...
 
-* <!-- MAGETWO-73419 --> When sorting simple products with a required custom option, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
+*  <!-- MAGETWO-73419 --> When sorting simple products with a required custom option, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```
Comment (jeff-matthews) [inline guides/v2.2/release-notes/ReleaseNotes2.2.5EE.md]:
```suggestion
*  <!-- MAGETWO-73419 -->When sorting simple products with a required custom option, which catalog promo price rule is applied for, these products are sorted by a regular price instead disregarding the applied promo price.
```

**Pull request merged**